eng/build.ps1

```diff
@@ -214,6 +214,13 @@ if ($vs) {
     } else {
       # Search for the solution in coreclr
       $vs = Split-Path $PSScriptRoot -Parent | Join-Path -ChildPath "src\coreclr" | Join-Path -ChildPath $vs | Join-Path -ChildPath "$vs.slnx"
+      
+      if (-Not (Test-Path $vs)) {
+        $vs = Split-Path $PSScriptRoot -Parent | Join-Path -ChildPath "src\coreclr\tools\aot\$solution.slnx"
+        if (-Not (Test-Path $vs)) {
+          $vs = Split-Path $PSScriptRoot -Parent | Join-Path -ChildPath "src\coreclr\nativeaot\$solution.slnx"
+        }
```

Comment:
This one already works, doesn't it? Should be covered by line 216 above.

Reply:
ah, you're right - removed.